components/automate-ui/src/app/pages/project/confirm-apply-stop-modal/confirm-apply-stop-modal.component.ts

```diff
@@ -42,4 +47,10 @@ export class ConfirmApplyStopModalComponent implements OnChanges {
     this.progressPrefixText = `${Math.floor(percentageComplete * 100)}% complete`;
     this.progressSuffixText = `${durCountdown} until finished`;
   }
+
+  public updateButtonText(inProgress: boolean): void {
+    this.stopRulesButtonText = inProgress
+      ? 'Stopping Project Update...'
+      : 'Stop Updating Projects';
```

Comment:
Just double-checking -- this is the casing we want? Not `Stop updating projects` or anything? 😃 (Asking because _The Right Thing_ is never clear to me in this question)

Reply:
Button labels are title case, or that's the standard we are trying to achieve.

Reply:
Cool. Thanks.

I suppose another way to do this, which I think I'd slighly prefer, is to have
```html
<button>{{ inProgress ? 'Stopping Project Update...' : 'Stop Updating Projects' }}</button>
```
in the template. So, looking at the template, you'd **know** what could appear in that position. With `{{ stopRulesButtonText }}` in the template, you've got to go check. @msorens what do you think? 😃

Reply:
I am definitely in favor--will change.